LeaderboardBackend.Test/TestUsers.cs

```diff
@@ -60,16 +63,63 @@ public static async Task GetUser_Found()
 		Assert.AreEqual(createdUser, retrievedUser);
 	}
 
+	[Test]
+	public static async Task Register_BadRequest()
+	{
+		RegisterRequest[] requests = {
+			new() {},
+			new()
+			{
+				Username = ValidUsername,
+				Password = ValidPassword,
+				PasswordConfirm = "someotherpassword",
+				Email = ValidEmail,
+			},
+			new()
+			{
+				Username = ValidUsername,
+				Password = ValidPassword,
+				PasswordConfirm = ValidPassword,
+				Email = "whatisthis",
+			},
+			new()
+			{
+				Username = "B",
+				Password = ValidPassword,
+				PasswordConfirm = ValidPassword,
+				Email = ValidEmail,
+			},
+		};
+
+		foreach(RegisterRequest request in requests)
+		{
+			HttpResponseMessage registerResponse = await ApiClient.PostAsJsonAsync("/api/users/register", request, JsonSerializerOptions);
+			Assert.AreEqual(
+				HttpStatusCode.BadRequest, 
+				registerResponse.StatusCode, 
+				$"{request} did not produce BadRequest, produced {registerResponse.StatusCode}"
+			);
+		}
+	}
+
+	[Test]
+	public static async Task Me_Unauthorized()
+	{
+		HttpRequestMessage meRequest = new(HttpMethod.Get, "/api/users/me");
+		HttpResponseMessage meResponse = await ApiClient.SendAsync(meRequest);
+		Assert.AreEqual(HttpStatusCode.Forbidden, meResponse.StatusCode);
+	}
+
 	[Test]
 	public static async Task FullAuthFlow()
```

Comment:
Gotta make a point to:
1. Implement logging out in the controller; and
2. Test logging out in this test method as well.